app/templates/components/index/index-action-buttons.hbs

```diff
@@ -1,13 +1,13 @@
 <div class="row justify-content-around index-action-buttons">
   <div class="info-holder col-lg-4 col-md-6 col-sm">
     <p class="text-left"> {{t 'component.index.actionButtons.member'}} </p>
-    <a class="btn btn-lg frontpage-button" href="https://docs.google.com/forms/d/e/1FAIpQLSdBkdRWrvv98ZX_JTOMtf9IDtIujHD6SZzxcg5S6JX0slb_zg/viewform?usp=sf_link" target="_blank">
+    <a class="btn btn-lg frontpage-button" href="https://forms.gle/R4WYzbWameD6wN817" target="_blank">
       {{t 'component.index.actionButtons.memberButton'}}
     </a>
   </div>
   <div class="info-holder col-lg-4 col-md-6 col-sm">
     <p class="text-left"> {{t 'component.index.actionButtons.room'}} </p>
-    <a class="btn btn-lg frontpage-button" href="https://docs.google.com/forms/d/e/1FAIpQLSd69A3IpEvUJZAkXWxGU6sRjonoSPCq8rrpslNYIXFqKvSHRg/viewform?usp=sf_link" target="_blank">
+    <a class="btn btn-lg frontpage-button" href="https://forms.gle/BcJsvEUqp5PJVzLj9" target="_blank">
```

Comment:
This form misses the data consent part

Reply:
Oh yeah it does... The current form also misses it. I added it on the new form. Well spotted!